indico/modules/events/registration/controllers/management/fields.py

```diff
@@ -29,6 +30,7 @@ class Meta:
     description = fields.String(load_default='')
     is_required = fields.Bool(load_default=False)
     input_type = fields.String(required=True, validate=not_empty)
+    retention_period = fields.TimeDelta(load_default=None)
```

Comment:
careful about the type here; check #5283 where this is passed as a number indicating the weeks